adagrams/game.py

```diff
@@ -1,11 +1,93 @@
+import random 
+
+available_letters = {
+    'A': [9, 1],
+    'B': [2,3], 
+    'C': [2, 3], 
+    'D': [4, 2], 
+    'E': [12, 1], 
+    'F': [2, 4], 
+    'G': [3, 2], 
+    'H': [2, 4], 
+    'I': [9, 1], 
+    'J': [1, 8], 
+    'K': [1, 5], 
+    'L': [4, 1], 
+    'M': [2, 3], 
+    'N': [6, 1], 
+    'O': [8, 1], 
+    'P': [2,3], 
+    'Q': [1,10], 
+    'R': [6,1], 
+    'S': [4,1], 
+    'T': [6,1], 
+    'U': [4,1], 
+    'V': [2,4], 
+    'W': [2,4], 
+    'X': [1,8], 
+    'Y': [2,4], 
+    'Z': [1,10]
+}
 def draw_letters():
-    pass
+    """Draws a hand of 10 letters"""
+    
+    letter_hand = []
+
+    while len(letter_hand) < 10:
+        picked_letter = random.choice(list(available_letters.keys()))
+        # check that letter hasn't exceeded available letter count
+        if letter_hand.count(picked_letter) < available_letters[picked_letter][0]:
+            letter_hand.append(picked_letter)
+    
+    return letter_hand    
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    """Checks that word can be made with letters in letter_bank"""
+    letter_bank_copy = letter_bank[:]
```

Comment:
Great job making a copy here. For example, you are removing letters from the original letter_bank, but the function returns False, meaning the word can't be made. Well, if you remove all the letters from the list, now the player's hand is empty! and they can't play the round anymore.